optout/src/main/java/gov/cms/ab2d/optout/ConsentConverterService.java

```diff
@@ -0,0 +1,19 @@
+package gov.cms.ab2d.optout;
+
+import gov.cms.ab2d.common.model.Consent;
+
+import java.util.Optional;
+
+public interface ConsentConverterService {
+
+    /**
+     * Given a line from a file, and its corresponding line number,
+     * converts the string into a Consent onject that can be persisted in the database.
+     *
+     * @param line
+     * @param lineNum
+     *
+     * @return (optional) Consent
+     */
+    Optional<Consent> convert(String line, int lineNum);
```

Comment:
We should probably not introduce the `lineNum` parameter if it is used merely for logging. Let's drop it and log the line number from the "caller" class, since it is available there, too.

Reply:
removed lineNum